app/controllers/authenticate_controller.rb

```diff
@@ -70,7 +70,7 @@ def authenticate
       authenticators: installed_authenticators,
       enabled_authenticators: Authentication::InstalledAuthenticators.enabled_authenticators_str(ENV)
     )
-    render json: authn_token
+    render json: (params[:encoded] ?  authn_token : Base64.strict_encode64(authn_token.to_json))
```

Comment:
another option is to pass this param into the `Authenticate` command class and perform this encoding (if needed) there. This way we still have the logic inside the command class.

wdyt?

Reply:
it will also be easier to UT

Reply:
Agreed. Thanks.